File: app/src/main/java/com/example/themoviedb/person_image_screen/person_image_view/ImageAcvtivityInterface.java
package com.example.themoviedb.person_image_screen.person_image_view;

import android.content.Intent;

public interface ImageAcvtivityInterface {
    Intent getIntent();

    void sendPhotoStringToActivity(String photo);

    void savePhoto();

    void setBroadcast(Intent mediaScanIntent);
}

File: app/src/main/java/com/example/themoviedb/person_image_screen/person_image_view/ImageAcvtivityInterface.kt
package com.example.themoviedb.person_image_screen.person_image_view

import android.content.Intent

interface ImageAcvtivityInterface {

    fun sendPhotoStringToActivity(photo: String)

    fun savePhoto()

    fun setBroadcast(mediaScanIntent: Intent)

    fun getPermission()

    fun getPicturePath()
}
